hive-watcher-js/hive-watcher.js

```diff
@@ -0,0 +1,44 @@
+const client = new dhive.Client([
+    'https://api.openhive.network',
+    'https://api.hive.blog',
+    'https://api.hivekings.com',
+    'https://anyx.io',
+    'https://rpc.ausbit.dev',
+    'https://hive.roelandp.nl',
+    'https://api.c0ff33a.uk',
+    'https://api.deathwing.me',
+    'https://hive-api.arcange.eu',
+    'https://fin.hive.3speak.co',
+    'https://hived.emre.sh',
+    'https://techcoderx.com',
+    'https://rpc.ecency.com',
+    'https://hived.privex.io',
+    'https://api.pharesim.me'
+]);
+
+
+function handlePost(post) {
+    let operationType = post.op[0]
+    if (operationType !== 'custom_json')
+        return
+
+    let operation = post.op[1]
+    if (operation.id !== 'podping')
+        return;
+
+    let postJson = JSON.parse(operation.json)
+
+    let block = post.block
+    let timestamp = post.timestamp
+
+    const message = `Feed updated - ${timestamp} - ${block} - ${post.trx_id} - ${postJson["url"]}`;
+    console.log(message)
+
+    const list = document.getElementById("posts");
+    const item = document.createElement("li");
+    item.appendChild(document.createTextNode(message))
+    list.appendChild(item)
+}
+
+client.blockchain.getOperationsStream()
+    .on('data', (post) => handlePost(post));
```

Comment:
Nitpicking, but this can just be
.on('data', handlePost);

:)

Reply:
Thanks